### DISCUSSION TITLE
world generation issue after update scml package

### DISCUSSION THREAD

**Discussion by snippet2021** [yasserfarouk/scml#98] [one-shot-open-discussion]
I have issue after updating scml package yesterday.
When I run myagent.py of template ,I have some issues.

1. Line of the agent provided by the system changes among the Worlds even though it has the same config.(see picture)
2. Multiple of myagent appear in one world.
3. None of myagent appear in the tournament.
4. I think #96 is same issue

competitors = [MyAgent, RandomOneShotAgent, SyncRandomOneShotAgent]
![スクリーンショット 2022-05-17 172942](https://user-images.githubusercontent.com/92664206/168766771-d6e4cc26-f677-4859-98f2-a26fa4eccd20.png)
.

**Comment by yasserfarouk** [comment]
I confirm that the cause for this seems to be the same as #96 .
Will push a big fix asap
